docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -215,54 +213,112 @@ for more information about the environment variables.
 
 The index name to write events to. The default is
 +"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
-+"{beatname_lc}-{version}-2017.04.26"+). If you change this setting, you also
++"{beatname_lc}-{version}-{localdate}"+). If you change this setting, you also
 need to configure the `setup.template.name` and `setup.template.pattern` options
-(see <<configuration-template>>).
-
-When loading the pre-built Kibana dashboards via {beatname_uc},
-you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
+(see <<configuration-template>>). If you are using the pre-built Kibana
+dashboards, you also need to set the `setup.dashboards.index` option (see
+<<configuration-dashboards>>).
```

Comment:
This removes the clause again _When loading the pre-built Kibana dashboards via APM Server_ ... which should not be removed, as it is deprecated.